<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestsWindow</class>
 <widget class="QMainWindow" name="TestsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>getBalances</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>placeOrderSell</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>placeOrderBuy</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Withdraw</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>WithdrawInternal</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>getDepositsList</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>cancelOrder</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>getOpenOrders</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>getOrdersHistory</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Result</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tests options</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Exchange</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="targetExchange">
         <item>
          <property name="text">
           <string>Select exchange</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Order size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="orderSize">
         <property name="decimals">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Withdraw BTC amount</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="withdrawAmount">
         <property name="decimals">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Withdraw USDT amount</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="withdrawAmountUsd">
         <property name="decimals">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Start tests</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Withdraw target exchange</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="withdrawTargetExchange">
         <item>
          <property name="text">
           <string>Select exchange</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
